### PR TITLE
Fix unstable test in parallel execution

### PR DESCRIPTION
**Pull request by vmanilo**
Resolves https://github.com/Twingate/terraform-provider-twingate/issues/167

## Changes
- fix acc test

### CODE REVIEW

**Comment by vmanilo** [comment]
I'll revert all CI changes, for now just needed to check acctests on my fork

File: .github/workflows/smoketests.yml [hunk 1]
```diff
@@ -54,7 +54,7 @@ jobs:
     timeout-minutes: 15
     strategy:
       fail-fast: false
-      max-parallel: 1
+#      max-parallel: 1
```
Comment (ekampf) [inline .github/workflows/smoketests.yml:57]: If its unecessary lets delete it instead of commenting out

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -146,69 +146,100 @@ jobs:
     timeout-minutes: 15
     strategy:
       fail-fast: false
-      max-parallel: 1
+#      max-parallel: 1
```
Comment (ekampf) [inline .github/workflows/ci.yml:149]: Delete instead of comment

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -146,69 +146,100 @@ jobs:
...
+        uses: actions/checkout@v3
+
+      - name: Set up Go
+        uses: actions/setup-go@v3
+        with:
+          go-version-file: 'go.mod'
+          cache: true
+        id: go
+
+      - name: Get dependencies
```
Comment (ekampf) [inline .github/workflows/ci.yml:225]: Need to add the caching statements (see lines 172-185) here too

Comment (vmanilo) [reply .github/workflows/ci.yml:239]: thank you, added to smoke tests as well

File: .github/workflows/smoketests.yml (hunk 1 above)
Comment (vmanilo) [reply .github/workflows/smoketests.yml]: sure, deleted

File: .github/workflows/ci.yml (hunk 2 above)
Comment (vmanilo) [reply .github/workflows/ci.yml]: deleted

**Review by ekampf — APPROVED** [review]

**Review by ekampf — APPROVED** [review]

**Pull request merged**